templates/takeovers/_template.html

```diff
@@ -1,14 +1,39 @@
-<div class="row row-hero">
-	<div class="box no-border">
-		<h1></h1>
-	</div><!-- /.box -->
-</div><!-- / .row -->
-
-<div class="row equal-height">
-	<div class="six-col box box-highlight">
-		
-	</div><!-- /.six-col -->
-	<div class="six-col last-col box box-highlight">
-		
-	</div><!-- /.six-col -->
-</div><!-- / .row -->
+<section lang="{% if lang %}{{ lang }}{% else %}en{% endif %}" lang="fr" data-lang="{% if locale %}{{ locale }}{% else %}en_GB{% endif %}" class="{% if class %}{{ class }}{% else %}p-takeover{% endif %} js-takeover">
+  <div class="row u-equal-height">
+    <div class="{% if equal_cols %}col-6{% else %}col-7{% endif %}">
+      <h1>
+        {{ title }}
+      </h1>
+      {% if subtitle %}<h4>
+        {{ subtitle }}
+      </h4>{% endif %}
+      <div class="u-hide--small">
+        <p>
+          <a href="{{ url }}" class="{% if cta_class %}{{ cta_class }}{% else %}p-button--positive{% endif %}">{% if url == "http" %}<span class="p-link--external">{% endif %}{{ cta }}{% if url == "http" %}</span>{% endif %}
+          </a>
+          <p>
+        <p>
+          <a href="{{ url2 }}" {% if url3 == "http" %}class="p-link--external"{% endif %}>
+            {{ cta2 }}
+          </a>
+          <p>
+      </div>
+    </div>
+    <div class="{% if equal_cols %}col-6 {% else %}col-5{% endif %} u-align--center u-vertically-center p-takeover__rightcol">
+      <img src="{% if 'http' not in image %}https://assets.ubuntu.com/v1/{% endif %}{{ image }}" alt="" {% if image_style %}style="{{ image_style }}"{% endif %}>
+    </div>
+    <div class="u-hide--medium u-hide--large">
+      <p>
+        <a href="{{ url }}" class="{% if cta_class %}{{ cta_class }}{% else %}p-button--positive{% endif %}">{% if 'http' not in url %}<span class="p-link--external">{% endif %}{{ cta }}{% if 'http'  in url %}</span>{% endif %}
+        </a>
+      <p>
+      {% if url2 %}
+      <p>
+        <a href="{{ url2 }}" {% if 'http' not in url2 %}class="p-link--external"{% endif %}>
```

Comment:
Similar question to above - if `http` were not present, wouldn't that suggest a relative path i.e. not external?

Reply:
good catch, fixed